field_test.go

```diff
@@ -162,7 +162,7 @@ func TestFieldConstructors(t *testing.T) {
 }
 
 func TestStackField(t *testing.T) {
-	f := Stack("stacktrace")
+	f := stack("stacktrace", 0)
```

Comment:
Actually on second thought - I think we actually have to do the thing you said with strings.Contains, because Stack can be called in different ways, which is the problem here.